Projects/IntelligentObjectAvoider/README.md

```diff
@@ -17,6 +19,19 @@ Generally, `pip` is associated for `Python 2` applications and `pip3` is associa
 3. Plug in the distance sensor into any of the 2 `I2C` ports of the GoPiGo3 just like in the below photo.
 4. Run the actual script. Use only `Python 3` as `Python 2` is not supported.
 
+## Running It
+
+In order to run it, use the following command:
+```
+python3 robot.py
+```
+
+Depending on how your system is set, `Python 3` may direct to `python` executable or `python3` - most likely, it will point to `python3`.
```

Comment:
Now this one is confusing as `Python 3` is not a command. So how could it direct to anything?

Reply:
I'm removing the backticks now for Python 3.